dali/operators/decoder/nvjpeg/decoupled_api/nvjpeg_memory.h

```diff
@@ -23,12 +23,35 @@ namespace dali {
 
 namespace nvjpeg_memory {
 
+/**
+ * @brief Returns a buffer of at least the requested size, preferrably from the preallocated pool
+ *        If no buffer that satisfies the requested arguments, an allocation will take place
```

Comment:
The second sentence looks a bit weird.
```suggestion
 *        If no buffer  that satisfies the requested arguments already exists in the pool, an allocation will take place
```